terraform/040-id-broker/vars.tf

```diff
@@ -172,10 +172,57 @@ variable "desired_count" {
   default = "1"
 }
 
+variable "email_signature" {
+  type    = "string"
+  default = ""
+}
+
+variable "password_forgot_url" {
+  type = "string"
+}
+
 variable "password_profile_url" {
   type = "string"
 }
 
+variable "support_email" {
+  type = "string"
+}
+
+variable "support_name" {
+  type = "string"
+}
+
 variable "help_center_url" {
   type = "string"
-}
\ No newline at end of file
+}
+
+variable "send_invite_emails" {
+  type    = "string"
+  default = "true"
```

Comment:
ID Broker uses a default of `false` for this one. Any reason we would want to change that to `true` like this?

Reply:
yeah I suppose we can default that to false and then override when we're ready versus the reverse